app/src/main/java/com/example/bigowlapp/activity/BigOwlActivity.java

```diff
@@ -59,6 +60,10 @@ public boolean onMenuItemClick(MenuItem item) {
         if (item.getItemId() == R.id.overflow_home) {
             finish();
             startActivity(new Intent(this, HomePageActivity.class));
+        } else if (item.getItemId() == R.id.overflow_wiki_page_manual) {
+            final String WIKI_PAGE_MANUAL_URI = "https://github.com/Landry333/Big-Owl/wiki/Manual";
+            Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(WIKI_PAGE_MANUAL_URI));
```

Comment:
Is it possible to store this in strings.xml?

Reply:
Yep, and it's done.